google/cloud/forseti/services/cli.py

```diff
@@ -304,10 +304,18 @@ def define_scanner_parser(parent):
         title='action',
         dest='action')
 
-    action_subparser.add_parser(
+    run_scanner_parser = action_subparser.add_parser(
         'run',
         help='Run the scanner')
 
+    run_scanner_parser.add_argument(
+        '--scanner',
```

Comment:
Why are we going with `--scanner`?  Is it because it will support choices?

Reply:
yes, and alternatively, we could use a flag just dedicated to EPAS. What would you recommend?

Reply:
Okay, let's stay with `--scanner`.  :)